.github/workflows/.tests-matrix.yaml

```diff
@@ -75,7 +75,7 @@ jobs:
   pytest-export:
     name: pytest (poetry-plugin-export)
     runs-on: ${{ inputs.runner }}
-    if: inputs.run-pytest-export
+    if: ${{ inputs.run-pytest-export && contains(github.event.pull_request.labels.*.name, 'test poetry export')}}
```

Comment:
The name of the label is not consistent with our naming scheme (no spaces, but slashes). Maybe, just `test/export`?